vscode-extension/package.json

```diff
@@ -48,6 +48,24 @@
             "workspace_path": {
               "type": "string",
               "description": "Absolute path to the workspace root"
+            },
+            "workflow_mode": {
+              "type": "string",
+              "description": "Workflow mode selection: 'full' (all stages), 'minimal' (core stages only), or 'custom' (explicit stages)",
+              "enum": ["full", "minimal", "custom"]
+            },
+            "review_strategy": {
+              "type": "string",
+              "description": "Review strategy: 'prs' (intermediate PRs) or 'local' (single branch). Does not affect which prompt files are generated.",
```

Comment:
If it doesn't effect which prompt files are generated then why is it a param? If no good reason, remove param and check for any other instances of unnecessary parameters across the code, and report before making broader changes.